usagov/vote-gov-drupal
# VOTE-429: Add config and po files from Amharic.

## Jira ticket

[VOTE-429](https://cm-jira.usa.gov/browse/VOTE-429)

## Description

Add PO and Config files to enable and provide string translations for Amharic.

## Deployment and testing

### Pre-deploy

n/a

### Post-deploy

1. Locally, run `lando retune -y` to import config files and string translations from PO file.

### QA/Test

1.  Visit http://vote-gov.lndo.site/admin/config/regional/language and confirm that Amharic language is listed.
2. Visit http://vote-gov.lndo.site/admin/config/regional/translate
3. Do a search using the following: Translation: Amharic, Search In: Only translated strings
4. Confirm that the search returns a list of translated strings for Amharic.

## Checklist for the Developer




## Checklist for the Peer Reviewers
